_posts/2023-06-23-run-and-debug-rust-verification-harnesses-natively-on-vscode.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title:  "Run and debug Rust verification harnesses natively on VS Code"
+---
+
+Kani is a verification tool that can help you systematically test properties about your Rust code.
+To learn more about Kani, check out the [Kani tutorial](https://model-checking.github.io/kani/kani-tutorial.html) and our [previous blog posts](https://model-checking.github.io/kani-verifier-blog/).
+
+In today’s blog post, we introduce the [Kani VS Code Extension](https://marketplace.visualstudio.com/items?itemName=model-checking.kani-vscode-extension), available on the VS Code marketplace.
+The extension automatically detects all harnesses within the package, and allows users to run and debug them directly in VS Code.
+To allow a more comfortable verification experience, Kani is now usable directly from the VS Code user interface.
```

Comment:
```suggestion
To allow a more comfortable verification experience, Kani is now usable from the VS Code user interface.
```
(wanted to avoid repeating "directly" again)